mongodb/README.md

```diff
@@ -0,0 +1,212 @@
+mongodb-kubernetes-operator-6d46dd4b74-xxxwh   1/1     Running   0          92s
+```
+
+Press Ctrl+C to cancel the command.
+
+Finally, deploy the MongoDB cluster:
+```
+kubectl apply -f mongodb-config.yaml
+```
+
+Check that the cluster is deploying:
+```
+watch kubectl get pods
+```
+Result:
+```
+Every 2.0s: kubectl get pods
+NAME        READY   STATUS    RESTARTS   AGE
+mongodb-0   2/2     Running   0          3m29s
+mongodb-1   2/2     Running   0          2m25s
+mongodb-2   1/2     Running   0          61s
+```
+
+## Create a New Database
+Now that you have a running MongoDB cluster, let's create a database and a collection. A collection in MongoDB NoSQL architecture is comparable to a table in a relational database.
+
+You are going to create a JSON document in the `shakespeare` database, within the `quotes` collection. This document has the following format:
+```json
+{
+  "publication": "Romeo and Juliet",
+  "text": "What\'s in a name? That which we call a rose by any other word would smell as sweet..."
+}
+```
+
+When deploying the `StatefulSet`, the Operator creates a MongoDB Replica Set, where one `Pod` holds the primary data set and the other ones hold replicas. Let's first verify this by running the following command:
+
+```
+kubectl exec -it mongodb-0 -- mongosh --username admin --password mongo --quiet --eval 'printjson(rs.status())'
+```
+
+Identify the primary MongoDB node. You should see a JSON Key/Value pair like below:
+```
+stateStr: 'PRIMARY'
+```
+
+The following MongoDB commands must be run from the primary node. Replace the `Pod` ordinal number accordingly so you can create the database and populate the collection.
+
+```
+kubectl exec -it mongodb-0 -- mongosh admin < /root/ondat/mongo.sh
+```
+
+Result:
+```
+...
+mongodb [direct: primary] admin> { ok: 1 }
+mongodb [direct: primary] admin> switched to db shakespeare
+mongodb [direct: primary] shakespeare> { ok: 1 }
+mongodb [direct: primary] shakespeare> {
+  acknowledged: true,
+  insertedId: ObjectId("6209a09540c27d9ba2907b76")
+}
+```
+
+Let's query the database to check that the document has been correctly created.
+```
+kubectl exec -it mongodb-0 -- mongosh admin < /root/ondat/checkmongo.sh
+```
+
+Result:
+
+```
+...
+mongodb [direct: primary] admin> { ok: 1 }
+mongodb [direct: primary] admin> switched to db shakespeare
+mongodb [direct: primary] shakespeare> [
+  {
+    _id: ObjectId("6209a09540c27d9ba2907b76"),
+    publication: 'Romeo and Juliet',
+    text: "What's in a name? That which we call a rose by any other word would smell as sweet..."
+  }
+]
+```
+
+All good! You have deployed a new database that contains a document. Now, Let's verify that the database is encrypted!
+
+## Verify Volume Encryption
+First, let's find the Ondat volumes associated with the MongoDB `Pods` by executing the following commands:
+```
+#Enable Ondat CLI first
+ kubectl port-forward svc/storageos -n storageos 5705 &
```

Comment:
```suggestion
```

Reply:
didn't add CLI container or deployment on that one, so keeping it as local.